.gitignore

```diff
@@ -65,3 +65,9 @@ coverage.xml
 !.gitignore
 .noseids
 Procfile
+
+# Local configuration file of udata
+udata.cfg
+
+# Local data directory
+data
```

Comment:
Same as @davidbgk for me. The default values should be in `.gitignore`. Custom/specific files for each of us should be in your `.git/ignore` file.
I can't remember, are `data` and `udata.cfg` the file created by default ?